src/tsserver/nodeServer.ts

```diff
@@ -688,6 +688,7 @@ namespace ts.server {
 
                 super({
                     host,
+                    fshost,
                     cancellationToken,
                     ...options,
                     typingsInstaller: typingsInstaller || nullTypingsInstaller,

```

Comment:
I am assuming typing installer part is for later ?..

Reply:
I think I missed something when adding `fshost` to `NodeTypingsInstaller`. 
I didn't pass `fshost` to objects that don't call filesystem operations, and NodeTypingsInstaller only calls `setImmediate` and `setTimeout` on its `host` reference, so I skipped it.

But from a high level, this doesn't make sense. How is the typings installer able to install things without fileystem access? Is it delegating to `npm install`, or something else that accesses the filesystem?

Reply:
Based on [your other comment](https://github.com/microsoft/TypeScript/pull/48497/files#r863156717), I think the right answer is: yes, let's leave it for a follow-up PR after I have a chance to talk to @mjbvz about ideas for a typings installer that communicates with vs code instead of running `npm install` itself.